test/mailer_notifier_test.exs

```diff
@@ -2,6 +2,8 @@ defmodule MailerNotifierTest do
   use ExUnit.Case
```

Comment:
I have the feeling we need to rename this file to `test/mailer_notifier/bamboo.exs` because this is only testing with this email library

(well, it's actually using the `FakeMailer` module but this is overriding the functions from Bamboo, I think)

However, we probably have some logic related to some of the tests that are not fully dependant on the email library, so it could be a task for a different PR.

Reply:
We had a discussion here about how to tackle this. Since we should test the same no matter which library we are using, the author of #71 came up with this solution:

https://github.com/wyeworks/boom/blob/78f2798795013ff3d0bd3f9a4c79c4baf180ddf7/test/mailer_notifier_test.exs#L48-L54

They also split the FakeMailer into two modules.

Let us know if you have any feedback about this.

Reply:
Right! I saw this PR later 😄 

Fair enough to me 👍